app/components/Checkbox/index.js

```diff
@@ -46,26 +46,23 @@ const Checkbox = ({
 
     {/* This is the control box with a check icon as children */}
     <ControlBox
-      borderWidth="1.5px" // TODO: should come from theme
-      size="14px" // TODO: should come from theme
-      borderColor={
-        isDisabled ? 'checkbox.disabled.borderColor' : 'checkbox.borderColor' // TODO: Based on theme.borderColor
-      }
+      size={0.875}
+      borderWidth={1.5}
```

Comment:
Let width come from theme too. Border width generally has definite set of values.

Reply:
Thanks for pointing out, I must have missed it with a confusion.